File: Src/API/WebApplication5/Model/regio.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    public class Regio
    {
        public int Id { get; set; }
        public string naam { get; set; }
        public List<Locatie> locaties { get; set; }
        public int tijd { get; set; }
        public string plaats { get; set; }
        
     /*   public regio(string naam, List<Locatie> locaties, int tijd)
        {
            this.naam = naam;
            this.locaties = locaties;
            this.tijd = tijd;
        }*/
    }
}

File: MobApp/app/src/main/java/be/ap/eaict/geocapture/Regio.java
package be.ap.eaict.geocapture;

import java.util.List;

public class Regio {
    public int Id;
    public String naam;
    public List<Locatie> locaties;
    public int tijd;

    public Regio(String naam, List<Locatie> locaties, int tijd)
    {
        this.naam = naam;
        this.locaties = locaties;
        this.tijd = tijd;
    }
}
